docs/ovms_quickstart.md

```diff
@@ -12,6 +12,8 @@ This guide uses a [Faster R-CNN with Resnet-50 V1 Object Detection model](https:
 
 > **Note**: - OpenVINO Model Server can run on Linux and macOS. For use on Windows, [WSL](https://docs.microsoft.com/en-us/windows/wsl/) is required.
 
+> **Note**: - OpenVINO Model Server can run on Linux and macOS. For use on Windows, [WSL](https://docs.microsoft.com/en-us/windows/wsl/) is required.
```

Comment:
This file changed on main so it should be rebased

Reply:
Has been rebased.